src/main/java/com/dre/brewery/configuration/files/Config.java

```diff
@@ -47,21 +46,15 @@ public class Config extends AbstractOkaeriConfigFile {
 
 
     @LocalizedComment("config.storage.header")
-    private Storage storage = new Storage();
-    @Getter @Setter
-    public static class Storage extends OkaeriConfig {
-        @LocalizedComment("config.storage.type")
-        private DataManagerType type = DataManagerType.FLATFILE;
-        @LocalizedComment("config.storage.database")
-        private String database = "brewery-data";
-        private String tablePrefix = "brewery_";
-        private String address = "localhost";
-        private String username = "root";
-        private String password = "";
-    }
-
-
-    // Maybe condense more of this into configuration sections
+    private ConfiguredDataManager storage = ConfiguredDataManager.builder()
```

Comment:
I like the previous solution more

Reply:
Could go back to it, but it adds extra bulk code in the storage area of BreweryX